README.md

```diff
@@ -2,27 +2,204 @@
+
+        [tls]
+        ca_cert =
+        cert =
+        key =
+        key_password =
+        tls_verify =
+
+6. Configure one or more Carbon Black Cloud Organizations
+
+    Example Organization
+
+        [OrgName1]
+        server_url = defense.conferdeploy.net
+        org_key = ABCD1234
+        custom_api_id = ABCDE12345
+        custom_api_key = ABCDEFGHIKLMNO1234567890
+        alerts_enabled = true
+        audit_logs_enabled = true
+
+    a. The `server_url` should match the hostname of your Carbon Black Cloud environment
+
+    b. The `org_key` can be found on the API Access page in the Carbon Black Cloud console from step 1
+
+    c. Use the CUSTOM API key from step 1
+
+    d. Enable the desired data you would like to send for the organization
+
+7. If you set `alerts_enabled` to `true` then you will need to configure one or more `alert_rules`
+
+    Each `alert_rules` is a separate request for alerts such that you can configure custom criteria for a desired usecase. See [Search Fields - Alert](https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/alert-search-fields/) for the fields marked **Searchable**.
+
+    Example Alert Rules
+
+        [[OrgName1.alert_rules]]
+        type = [ "WATCHLIST", "DEVICE_CONTROL" ]
+        minimum_severity = 7
+
+        [[OrgName1.alert_rules]]
+        type = [ "CB_ANALYTICS" ]
+        minimum_severity = 3
+
+    The key is the alert field you want to filter by and the value is a list of values you want to filter except `minimum_severity` which is a single integer. Each value is OR'd for a key and values are AND'd across keys e.g. `type:( WATCHLIST OR DEVICE_CONTROL) AND minimum_severity: 7`
+
+
+### Running cbc_syslog_forwarder
+
+The script `cbc_syslog_forwarder` is installed into the OS bin directory for easy access from any directory
+
+```
+>>> cbc_syslog_forwarder --help
+usage: cbc_syslog_forwarder [-h] [--log-file LOG_FILE] [-d] [-v] {poll,history,check} ...
+
+positional arguments:
+  {poll,history,check}  The action to be taken
+    poll                Fetches data from configured sources and forwards to configured output since last poll
+                        attempt
+    history             Fetches data from source(s) for specified time range and forwards to configured
+                        output
+    check               Check config for valid API keys with correct permissions
+
+options:
+  -h, --help            show this help message and exit
+  --log-file LOG_FILE, -l LOG_FILE
+                        Log file location
+  -d, --debug           Set log level to debug
+  -v, --verbose         Set log level to info
+```
+
+The `cbc_syslog_forwarder` poll command is designed to be executed in a cronjob for continual syslog forwarding
+
+```
+5  *  *  *  * root cbc_syslog_forwarer --log-file /some/path/cbc-syslog.log poll /some/path/my-config.toml
+```
+
+### Customer Support
```

Comment:
Syslog is supported, so should direct people to dev network and creating a case. Maybe the contact us form?
https://developer.carbonblack.com/contact